openassessment/xblock/staff_area_mixin.py

```diff
@@ -266,6 +266,12 @@ def render_staff_grade_form(self, data, suffix=''):  # pylint: disable=W0613
             submission_to_assess = staff_api.get_submission_to_assess(course_id, item_id, staff_id)
 
             if submission_to_assess is not None:
+                self.runtime.publish(self, 'openassessmentblock.get_submission_for_staff_grading', {
```

Comment:
Nit: I think this is poor naming by the XBlock API but the way this reads it seems to be the XBlock itself is being published. Maybe just add a comment to say that this is publishing an event.